bundle/compliance/cis_aws/rules/cis_1_19/rule.rego

```diff
@@ -0,0 +1,28 @@
+package compliance.cis_aws.rules.cis_1_19
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.data_adapter
+import future.keywords.every
+
+default rule_evaluation = false
+
+finding = result {
+	data_adapter.is_server_certificate
+
+	result := common.generate_result_without_expected(
+		common.calculate_result(rule_evaluation),
+		{"certificates": data_adapter.server_certificates},
+	)
+}
+
+is_expired(date) {
```

Comment:
Can you use `common.date_within_duration` instead?
for example, `common.date_within_duration(time.parse_rfc3339_ns(date), "0h")`